spec/models/notification_spec.rb

```diff
@@ -298,6 +296,18 @@
       let!(:notification) { create(:notification, user: user, notifiable: comment, action: "Reaction") }
       let(:sibling_reaction) { create(:reaction, reactable: comment, user: user3) }
 
+      before do
```

Comment:
why this?

Reply:
For the same reason mentioned here: https://github.com/thepracticaldev/dev.to/pull/5401#discussion_r364198021